client/src/pages/HomePage.js

```diff
@@ -27,18 +24,23 @@ class Home extends Component {
 
     render() {
         var display;
-        if(this.state.user){
+        if (this.state.user) {
             display = <p>Hello {this.state.user.firstName}!</p>
-        }else{
+        } else {
             display = <p>Hello there! <a href="/">Sign up here!</a></p>
         }
+
         return (
             <div>
-                <HomeNav/>
+                <HomeNav />
                 <Wrapper>
-                   {
-                       display
-                   }
+                    {display}
+                    <div className="userInfo">
+                        <img src={this.state.user.avatar} alt={this.state.user.firstName} width="150" height="150"/>
```

Comment:
Looks like a job for object destructing. 
```javascript
const { avatar, firstName } = this.state;
```
Then you can use ust `avatar` and `firstName` in your jsx.